src/Service.Tests/Configuration/ConfigurationTests.cs

```diff
@@ -414,6 +430,19 @@ public async Task TestSqlSettingPostStartupConfigurations()
             message.Headers.Add(AuthorizationResolver.CLIENT_ROLE_HEADER, POST_STARTUP_CONFIG_ROLE);
             HttpResponseMessage authorizedResponse = await httpClient.SendAsync(message);
             Assert.AreEqual(expected: HttpStatusCode.OK, actual: authorizedResponse.StatusCode);
+
+            // OpenAPI document is created during config hydration and
+            // is made available after config hydration completes.
+            HttpResponseMessage postConfigOpenApiDocumentExistence =
+                await httpClient.GetAsync($"/api/{OPENAPI_DOCUMENT_ENDPOINT}");
```

Comment:
could use the const: {DEFAULT_REST_PATH}

Reply:
updated usage to instead be GlobalSettings.DEFAULT_REST_PATH.